tests/test_bidi.py

```diff
@@ -26,6 +26,12 @@ async def websocket():
     async with websockets.connect(url) as connection:
         yield connection
 
+@pytest.fixture(autouse=True)
+async def before_each_test(websocket):
```

Comment:
This was needed because `_getOrCreateContext` was made async, and an event is always emitted. This will probably change again when event filtering is added.